### ISSUE TITLE
[TW-303] Ability for import command to update existing task

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#339]
_Kosta Harlan on 2013-10-04T15:46:56Z says:_ 

 If a UUID is present in the JSON data, it would be nice for `task import` to able to update the existing task. The current behavior is to skip importing the task. I would be fine with "update on import" as the only behavior for `task import`, but it could also be configurable, e.g. rc.import.update=true would have "update on import" behavior while rc.import.update=false would skip over JSON data with a UUID.

I haven't looked at the code, but this would probably also allow `task sync` to re-use some of the code for `task import` since it could handle both creating and updating a task.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-10-04T15:46:56Z
Modified: 2015-05-30T07:08:43Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2013-11-01T22:02:33Z says:_ 

 Good suggestion.  Thanks.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2013-11-03T14:30:32Z says:_ 

 I've been thinking about this, and believe it is important.  Moving up to 2.4.0.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-11-09T23:53:22Z says:_ 

 Pushing to 2.5.0.

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2015-05-30T07:08:43Z says:_ 

 Implemented in 9d6067e2fe35cdab4289b56eb14e593c8539099e.

Thank you for the suggestion Kosta!
